result/arp.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.1.0
+
+info:
+  title: ARP table models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Arp.Entries.Request:
+      description: >-
+        The request to retrieve ARP entries per IPv4 interface.
+      type: object
+      properties:
+        ipv4_names:
+          description: >-
+            The names of IPv4 interfaces for which ARP entries will be retrieved.
+            If no interface names are specified then the result set will contain all ARP entries.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Ipv4/properties/name"
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned.
+            The column ipv4_name will always be present in the returned result set.
+          type: array
+          items:
+            type: string
+            enum:
+              - ipv4_name
+              - neighbor_ip
+              - link_layer_address
+
+    Arp.Entries.Response:
+      description: >-
+        A container for ARP entries.
+      type: object
+      properties:
+        resolved:
+          description: >-
+            Contains the ARP entries where link-layer address of the neighbor is already known.
+          type: array
+          items:
+            $ref: './arp.yaml#/components/schemas/Arp.Entry'
+        unresolved:
+          description: >-
+            Contains the ARP entries where link-layer address learning of the neighbor is still pending.
```

Comment:
Contains the ARP entries for which link-layer address (MAC) learning of the neighbor is still pending.